lib/game_teams.rb

```diff
@@ -0,0 +1,37 @@
+class GameTeams
```

Comment:
GameTeams sounds awkward.  Consider changing to TeamsGames maybe?

Reply:
Yeah or maybe it should be all singular? Since this class is representing one instance of gameteam, haha very weird to say that